project/BuildSettings.scala

```diff
@@ -28,14 +31,32 @@ object BuildSettings {
       fork in Test := true   // Sometimes this causes sbt test to fail,
     )
 
-  def projectSettings(assembly: Boolean = false) = buildSettings ++
-    Seq(resolvers += Classpaths.typesafeReleases,
-        resolvers ++= socrataResolvers,
-        scalacOptions ++= Seq("-Xlint", "-deprecation", "-Xfatal-warnings", "-feature"))
+  def projectSettings = buildSettings ++
+    Seq(
+      resolvers += Classpaths.typesafeReleases,
+      resolvers ++= socrataResolvers,
+      scalacOptions ++= Seq("-Xlint", "-deprecation", "-Xfatal-warnings", "-feature")
+    )
+
+  lazy val socrataResolvers =
+    Seq(
+      "Open Source Geospatial Foundation Repository" at "http://download.osgeo.org/webdav/geotools",
+      "spray repo" at "http://repo.spray.io",
+      "velvia maven" at "https://dl.bintray.com/velvia/maven"
+    )
+
 
-  lazy val socrataResolvers = Seq(
-    "Open Source Geospatial Foundation Repository" at "http://download.osgeo.org/webdav/geotools",
-    "spray repo" at "http://repo.spray.io",
-    "velvia maven" at "https://dl.bintray.com/velvia/maven"
-  )
+  lazy val rootSettings =
+    Seq(
+      // do not publish (no geospace folder created)
+      publish := {},
+      // do not publish locally
+      publishLocal := {},
```

Comment:
Not exactly sure what this key does - it doesn't turn off the ability to publish the subprojects locally, right? We would still want to do that for local development, eg. if I was working in another repo that had a dependency on my local snapshot version of geospace-lib.